apps/rush-lib/assets/rush-init/rush.json

```diff
@@ -386,6 +386,43 @@
    */
   /*[LINE "HYPOTHETICAL"]*/ "hotfixChangeEnabled": false,
 
+  /**
+   * Specify an autoinstaller name, whose packages are used by the whole monorepo.
+   * It will be auto installed before each rush commands run.
+   *
+   * Example: assume you want to use 'common-packages' autoinstaller:
+   *   rush init-autoinstaller --name common-packages
+   *   cd common/autoinstallers/common-packages
+   *   pnpm install some-rush-plugins
+   *   rush update-autoinstaller --name common-packages
+   */
+  /*[LINE "HYPOTHETICAL"]*/ "commonAutoinstallerName": "common-packages",
```

Comment:
I wonder if "common" is too generic -- is there a use case for such a list of packages beyond the plugin framework?

It might tie the feature together better in the user's head if the `rushPlugins` section references packages installed in the `common/autoinstallers/plugins` folder by default.

Reply:
This is really a design choice, let me do some explanation on my thoughts here.

Some other monorepo tools(such as lerna) would like install a "monorepo-level" dependencies on repo top-level folder, while rush.js disencourage this behaviour of installing packages on top-level folder(which is good). 

And now, we are facing the problem on lacking of the package depdnencies used in a scope of "monorepo-level" in rush.js. This is `xxxAutoinstallerName` comes into rescue. With this autoinstaller, rush.js empowered to use third-party modules before execute core logic.

With the scope of this PR, yes, this `xxxAutoinstallerName` should be better named like `pluginsAutoinstallerName`: 'plugins'. But what if we need other features which encounter the same problem as plugins in the furture, I mean some features need third-party modules installed before execution. In that time, another `yyyAutoinstallerName` will show up in `rush.json`. So "common" was chosed here to avoid this situation. And, I admit it's kind of preconsideration.

And agian, i am open to rename it to `pluginsAutoinstallerName` if it's a better choice.

Reply:
Let me rename to `pluginsAutoinstallerName`.

Would you prefer to stay it in `rush.json` or a brand new file sth like `rush-plugins.json`